src/www/app/app.ts

```diff
@@ -550,10 +557,22 @@ export class App {
   //#region UI dialogs
 
   private showConfirmationDialog(message: string): Promise<boolean> {
-    // Temporarily use window.alert and window.confirm here
+    // Temporarily use window.confirm here
     return new Promise<boolean>(resolve => resolve(confirm(message)));
```

Comment:
We were only using `confirm` in electron, I think it would be better to see the behavior on mobile platforms?

Reply:
Right, and more importantly, whether it works on mobile?

Reply:
Very useful!